tests/test_pedantic.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+import pytest
+
+from client_code.pedantic import in_list, validated
+
+__version__ = "0.0.1"
+
+valid_items = ["one", "two"]
+
+
+@in_list("item", valid_items)
+@validated
```

Comment:
What about?

```python
@validate(item=InList(items), ...)

```

Reply:
I thought this gave a little more control - you might forseeably want the individual validators but not need the overall is_valid method.

Easily persuaded otherwise!

Reply:
i.e. the is_valid method is only really useful server side to check that client code hasn't been messed about with and sent an invalid instance.

Reply:
Yeah - I'll see what I can do...

Reply:
It looks like skulpt doesn't add the class attribute anyway! Back to the drawing board...

Reply:
What do you mean?

Reply:
All fixed and working now